packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -414,6 +471,7 @@ export const useTabStyles_unstable = (state: TabState): TabState => {
     state.contentReservedSpaceClassName = mergeClasses(
       reservedSpaceClassNames.content,
       contentStyles.base,
+      size === 'large' && contentStyles.large,
```

Comment:
Sorry, one change I just caught -- this looks like it should actually be `contentStyles.largeSelected`, and needs to come after the `contentStyles.selected` classname below, so the large font-size overrides the regular selected font-size

Reply:
Great catch!